src/granite_orm/table.cr

```diff
@@ -45,5 +45,13 @@ module Granite::ORM::Table
     def self.primary_name
       @@primary_name
     end
+
+    def self.quoted_table_name
+      @@adapter.quote_table_name(table_name)
+    end
+
+    def self.quoted_column_name(column_name)
```

Comment:
Since this method is providing an action, it should be named `quote_column_name` or perhaps even just `quote_column`.